engine/src/main/java/org/hibernate/validator/spi/properties/ConstrainableExecutable.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.spi.properties;
+
+import java.lang.reflect.Type;
+
+/**
+ * A class that describes JavaBeans method. Is used for determining if a given
```

Comment:
It seems odd to explicitly mention "JavaBeans" here. Isn't the point of it exactly to go beyond the notion of JavaBeans? "Descriptor for a method of a Java class" might be a better alternative.

Reply:
I removed a couple in one of my latest commits but apparently missed this one. Fixed.